pallets/anchors/src/lib.rs

```diff
@@ -66,7 +66,7 @@ type BalanceOf<T> =
 /// This is the maximum expected time for document consensus to take place between a pre-commit of an anchor and a
 /// commit to be received for the pre-committed anchor. Currently we expect to provide around 80 mins for this.
 /// Since our current block time as per chain_spec.rs is 6s, we set this to 80 * 60 secs / 6 secs/block = 800 blocks.
-const PRE_COMMIT_EXPIRATION_DURATION_BLOCKS: u32 = 800;
+pub const PRE_COMMIT_EXPIRATION_DURATION_BLOCKS: u32 = 800;
```

Comment:
Why?

Reply:
It is referred from a public doc, as a value the user should know. Or we do this or we need to rewrite how to explain that to the user from `pre_commit()` docs without referring this const